proto/confidentialtransfers/query.proto

```diff
@@ -12,7 +12,7 @@ option go_package = "github.com/sei-protocol/sei-chain/x/confidentialtransfers/t
 service Query {
   rpc GetCtAccount (GetCtAccountRequest) returns (GetCtAccountResponse) {
     option (google.api.http) = {
-      get: "/seichain/confidentialtransfers/account/{address}/{denom}"
+      get: "/seichain/confidentialtransfers/account/{address}/denom"
```

Comment:
I don't think we've implemented the rest route for this yet, we would have to take special care to add the denom as a parameter right?

Reply:
this will be added automatically